recipes/iot-services/iotagent-json/config.js

```diff
@@ -0,0 +1,36 @@
+var config = {};
+
+config.mqtt = {
+    host: 'mosquitto',
+    port: 1883,
+    thinkingThingsPlugin: true
+};
+
+config.iota = {
+    logLevel: 'DEBUG',
+    timestamp: true,
+    contextBroker: {
+        host: 'orion',
+        port: '1026'
+    },
+    server: {
+        port: 4041
+    },
+    deviceRegistry: {
+        type: 'mongodb'
+    },
+    mongodb: {
+        host: 'mongo',
+        port: '27017',
```

Comment:
anyway to make this an environment variable? it is supposed to be deployed in a different recipe.